### ISSUE TITLE
jsonrpc module "parity" not available

### ISSUE THREAD

**Issue by amexperts** [paritytech/parity#8965]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Which Parity version?**: Parity/v1.11.4-beta-cc44ae9-20180619/x86_64-linux-gnu/rustc1.26.1
> - **Which operating system?**: Linux
> - **How installed?**: via installer
> - **Are you fully synchronized?**: yes
> - **Which network are you connected to?**: ethereum
> - **Did you try to restart the node?**: yes

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

I have started parity with `parity --datadir /mnt/parity --jsonrpc-apis web3,eth,pubsub,net,parity,parity_pubsub,traces,rpc` also tried just `jsonrpc-apis safe` - when I then do geth attach and try to use the parity submodule it is not available. eth, web3 are though.

looking here: https://wiki.parity.io/JSONRPC
It looks like it should be.. I am trying to look at pending transactions. Thanks

**Comment by roninkaizen** [comment]
in detail you try to call this
https://wiki.parity.io/JSONRPC-parity-module.html#parity_pendingtransactions ?
the call can be done with curl -
why are you using "geth attach" in your configuration?
-> as you tried the call it without, does it work like intended?

**Comment by amexperts** [comment]
Yes, I can call with curl no problem. I was using geth attach as a convenient way for testing.. ie eth.syncing works nicely - eth being one of the JSONRpc modules, so I guess not all of the modules are exposed in the same way :)
